### PR TITLE
[add] - auth layout

### PR DESCRIPTION
**Pull request by gideonnnalue**
Desc - Added layout for authentication pages which are to be used across authentication pages (signup and signin).

### CODE REVIEW

File: src/components/auth/components/AuthLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            <Grid item md={6} className="left">
+                {authLeftInfo()}
+                <div className="l-authLayout__brandDesc">
+                    <div>{siteDesc1}</div>
+                </div>
+            </Grid>
+            <Grid item md={6} className="right">
+                {props.children}
+            </Grid>
+        </Grid>
```
Comment (enyeInc) [inline src/components/auth/components/AuthLayout.jsx]: Why do you have nested `Grid`s?

File: src/constants/siteData.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const siteName = 'health-e';
+const siteDesc1 =
+	'Connect with friends around the world with great health topics';
+const user = 'User';
+const medical = 'Medical';
+const signup = 'Sign up';
+
+export { siteName, siteDesc1, user, medical, signup };
```
Comment (enyeInc) [inline src/constants/siteData.js]: there should not be a `constants` folder. Each component has its own `constants.js` file into its folder hierarchy. Reference the [Enye Conventions](https://github.com/enyeInc/Engineering-Conventions#file-structure) to see how you should do this

File: src/sass/abstract/_mixin.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  1800px:     Big desktop
+  
+  $breakpoint argument choices
+  -phone
+  -tab-port
+  -tab-land
+  -big-desktop
+  
+  1em - 16px
+  */
```
Comment (enyeInc) [inline src/sass/abstract/_mixin.scss]: This is great commenting

**Comment by enyeInc** [comment]
@gideonnnalue code is ready to be merged, you just have to follow our file structure conventions when it comes to `constants.js` files

File: src/components/auth/components/AuthLayout.jsx (hunk 1 above)
Comment (gideonnnalue) [reply src/components/auth/components/AuthLayout.jsx]: Saw it written in the material ui documentation that way

File: src/sass/abstract/_mixin.scss (hunk 3 above)
Comment (gideonnnalue) [reply src/sass/abstract/_mixin.scss]: Thank you

**Pull request closed without merging**